warehouse/models/intermediate/gtfs/int_gtfs_schedule__stop_times_grouped.sql

```diff
@@ -3,19 +3,51 @@
 WITH
 
 dim_stop_times AS (
-
     SELECT * FROM {{ ref('dim_stop_times') }}
-
 ),
 
-int_gtfs_schedule__stop_times_grouped AS (
+stops AS (
+    SELECT
+        feed_key,
+        stop_id,
+        stop_timezone_coalesced,
+        COUNT(*) AS ct
+    FROM {{ ref('dim_stops') }}
+    WHERE stop_id IS NOT NULL
+    GROUP BY 1, 2, 3
+    -- we can have duplicate stop IDs within a given feed (this is not valid, but happens)
+    -- just keep the most common time zone (very unlikely to have same stop ID but different time zone)
+    QUALIFY RANK() OVER (PARTITION BY feed_key, stop_id ORDER BY ct DESC) = 1
+),
 
+stops_times_with_tz AS (
     SELECT
+        dim_stop_times.*,
+        COALESCE(FIRST_VALUE(stop_timezone_coalesced)
+            OVER (PARTITION BY feed_key, trip_id
+                ORDER BY stop_sequence
+                ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING), feed_timezone) AS trip_start_timezone,
```

Comment:
They include it in [the examples](https://cloud.google.com/bigquery/docs/reference/standard-sql/navigation_functions#first_value) so I included it here. But I just did a test and I think you may not need it if you're using ascending sort with first value but you do it you're using last value with descending sort because: 
<img width="936" alt="image" src="https://user-images.githubusercontent.com/55149902/234069257-a919c28f-8a9e-4e25-826d-003e68b9db1b.png">

TLDR, seems safer / fine to include it?

Reply:
Ok cool, that link does specify aggregation but I think it's fine

Reply:
Pedant mode the screenshot specifies aggregation functions but the actual link is to the navigation functions page where they do include it, and I manually verified that 

```
WITH finishers AS
 (SELECT 'Sophia Liu' as name,
  TIMESTAMP '2016-10-18 2:51:45' as finish_time,
  'F30-34' as division
  UNION ALL SELECT 'Lisa Stelzner', TIMESTAMP '2016-10-18 2:54:11', 'F35-39'
  UNION ALL SELECT 'Nikki Leith', TIMESTAMP '2016-10-18 2:59:01', 'F30-34'
  UNION ALL SELECT 'Lauren Matthews', TIMESTAMP '2016-10-18 3:01:17', 'F35-39'
  UNION ALL SELECT 'Desiree Berry', TIMESTAMP '2016-10-18 3:05:42', 'F35-39'
  UNION ALL SELECT 'Suzy Slane', TIMESTAMP '2016-10-18 3:06:24', 'F35-39'
  UNION ALL SELECT 'Jen Edwards', TIMESTAMP '2016-10-18 3:06:36', 'F30-34'
  UNION ALL SELECT 'Meghan Lederer', TIMESTAMP '2016-10-18 3:07:41', 'F30-34'
  UNION ALL SELECT 'Carly Forte', TIMESTAMP '2016-10-18 3:08:58', 'F25-29'
  UNION ALL SELECT 'Lauren Reasoner', TIMESTAMP '2016-10-18 3:10:14', 'F30-34')
SELECT name,
  FORMAT_TIMESTAMP('%X', finish_time) AS finish_time,
  division,
  FORMAT_TIMESTAMP('%X', fastest_time) AS fastest_time,
  TIMESTAMP_DIFF(finish_time, fastest_time, SECOND) AS delta_in_seconds
FROM (
  SELECT name,
  finish_time,
  division,
  LAST_VALUE(finish_time)
    OVER (PARTITION BY division ORDER BY finish_time DESC
    ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING) AS fastest_time
  FROM finishers)
```
Returns a different result than 
```
WITH finishers AS
 (SELECT 'Sophia Liu' as name,
  TIMESTAMP '2016-10-18 2:51:45' as finish_time,
  'F30-34' as division
  UNION ALL SELECT 'Lisa Stelzner', TIMESTAMP '2016-10-18 2:54:11', 'F35-39'
  UNION ALL SELECT 'Nikki Leith', TIMESTAMP '2016-10-18 2:59:01', 'F30-34'
  UNION ALL SELECT 'Lauren Matthews', TIMESTAMP '2016-10-18 3:01:17', 'F35-39'
  UNION ALL SELECT 'Desiree Berry', TIMESTAMP '2016-10-18 3:05:42', 'F35-39'
  UNION ALL SELECT 'Suzy Slane', TIMESTAMP '2016-10-18 3:06:24', 'F35-39'
  UNION ALL SELECT 'Jen Edwards', TIMESTAMP '2016-10-18 3:06:36', 'F30-34'
  UNION ALL SELECT 'Meghan Lederer', TIMESTAMP '2016-10-18 3:07:41', 'F30-34'
  UNION ALL SELECT 'Carly Forte', TIMESTAMP '2016-10-18 3:08:58', 'F25-29'
  UNION ALL SELECT 'Lauren Reasoner', TIMESTAMP '2016-10-18 3:10:14', 'F30-34')
SELECT name,
  FORMAT_TIMESTAMP('%X', finish_time) AS finish_time,
  division,
  FORMAT_TIMESTAMP('%X', fastest_time) AS fastest_time,
  TIMESTAMP_DIFF(finish_time, fastest_time, SECOND) AS delta_in_seconds
FROM (
  SELECT name,
  finish_time,
  division,
  LAST_VALUE(finish_time)
    OVER (PARTITION BY division ORDER BY finish_time DESC
) AS fastest_time
  FROM finishers)
```